### ISSUE TITLE
Documentation Suggestion

### ISSUE THREAD

**Issue by tjdevries** [hrsh7th/nvim-cmp#99]
I'd suggest removing these:

```
      ['<C-p>'] = cmp.mapping.select_prev_item(),
      ['<C-n>'] = cmp.mapping.select_next_item(),
```

from the suggested mappings, since this will make `cmp` override `C-n` and `C-p` just to run those same keystrokes again. This messed up some stuff in telescope for me because `cmp` applied the mappings after telescope (at least that's what it seemed like to me) and then I could not easily scroll up and down using my normal keybinds for telescope

**Comment by tjdevries** [comment]
Yes, the main problem is that if you have a keymap set to do something with `<c-n>`, then `cmp` will override the `<c-n>` keymap for that buffer.

This means that it will just send the default `<c-n>` instead of whatever you originally mapped it to.

At least that is what was happening for me

**Comment by hrsh7th** [comment]
nvim-cmp has an effort to support already mapped keys.

1. The user install `tpope/vim-endwise`
2. `tpope/vim-endwise` add the mapping to `<CR>` automatically.
3. The user set `mapping = { ['<CR>'] = cmp.mapping.confirm() } ` up too

In this case, `nvim-cmp` overwrite `<CR>` mapping and do the following.

First, nvim-cmp try to run `cmp.confirm()` that returns `confirmation invoked or not`.
Then, nvim-cmp's built-in mapping will invoke `fallback()` function if confirmation does not invoked.
The `fallback` function will feed `<CR><Plug>DiscretionaryEnd` in this case.

But I think you are completely right. We haven't to re-map `<C-n>` and `<C-p>` because it is the same as vim's native key mapping.

I just worried why this mapping breaks telescope at the moment.

I've removed it from README.md.
